### ISSUE TITLE
Add a setting screen

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-calendar#424]
Please add to this ticket every default settings you would like to be able to alter : 
 - default view (month on week) ;
 - set business hours ;
 - set a default reminder (for example by email one day before) ;

**Comment by frankrousseau** [comment]
Yep we could get inspiration from the one of Sunrise, it has all the good settings.
